core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -180,7 +180,6 @@ public long getDataTimestamp() {
         }
     }
 
-    @Restricted(NoExternalUse.class)
```

Comment:
Since this becomes part of the public API could you document it?

Reply:
addressed in 9d642ebd8c08d8642ef3a433f1a4035d01a1d4f7